skills/scribe/scripts/transcribe.py

```
@@ -0,0 +1,416 @@
+
+# Grouping thresholds for providers that return words rather than segments.
+SEG_MAX_GAP_S = 1.0     # a pause longer than this starts a new segment
+SEG_MAX_LEN_S = 20.0    # hard cap so one monologue doesn't become one blob
+
+TIMEOUT = httpx.Timeout(connect=30.0, read=1800.0, write=1800.0, pool=30.0)
+
+
+# ---------------------------------------------------------------- helpers
+
+def duration_s(path):
+    out = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)